spec/02-integration/04-admin_api/03-consumers_routes_spec.lua

```diff
@@ -247,6 +247,11 @@ describe("Admin API (#" .. strategy .. "): ", function()
           pages[i] = json
         end
       end)
+      it("not crashed by empty custom_id #KAG-867", function()
+        local res = client:get("/consumers?custom_id=")
```

Comment:
I'd like another test added with:

```
local res = client:get("/consumers?custom_id")
```

e.g. does that turn `custom_id` to `true` and then also error?